src/main/java/org/folio/circulation/domain/policy/LoanPolicy.java

```diff
@@ -94,7 +95,7 @@ public static LoanPolicy from(JsonObject representation) {
       errorWhenReachedRenewalLimit(loan, errors);
 
       if(errors.isEmpty()) {
-        return proposedDueDateResult.map(dueDate -> loan.renew(dueDate, getId()));
+        return proposedDueDateResult.map(dueDate -> loan.renew(dueDate, getId(),getName()));
```

Comment:
I find it easier to read method invocation when there are spaces after the comma separator between the parameters.

```suggestion
        return proposedDueDateResult.map(dueDate -> loan.renew(dueDate, getId(), getName()));
```